packages/apps/human-app/frontend/src/modules/worker/jobs/hooks/use-combine-pages.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react';
+
+export function useCombinePages<T>(
```

Comment:
It looks like a very common hook related to pagination that can be used also outside of jobs pages. Does it make sense to move it to shared hooks?